internal/runner/playwright.go

```diff
@@ -0,0 +1,155 @@
+package runner
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"slices"
+	"strings"
+
+	"github.com/buildkite/test-engine-client/internal/debug"
+	"github.com/buildkite/test-engine-client/internal/plan"
+	"github.com/kballard/go-shellquote"
+)
+
+var _ = TestRunner(Playwright{})
+
+type Playwright struct {
+	RunnerConfig
+}
+
+func (p Playwright) Name() string {
+	return "Playwright"
+}
+
+func NewPlaywright(p RunnerConfig) Playwright {
+	if p.TestCommand == "" {
+		p.TestCommand = "npx playwright test"
+	}
+
+	if p.TestFilePattern == "" {
+		p.TestFilePattern = "**/{*.spec,*.test}.{ts,js}"
+	}
+
+	if p.TestFileExcludePattern == "" {
+		p.TestFileExcludePattern = "**/node_modules"
```

Comment:
🤔 so by default we exclude node modules? If the customer specifies an exclude pattern, should we still exclude the node modules as well?

Reply:
Discussed in sync, we will exclude `**/node_modules` as built-in feature, without having to set it through exclude pattern. I will open a separate PR for this.